.github/workflows/windows.yml

```diff
@@ -8,6 +8,7 @@ jobs:
     name: Windows Serial
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
This will run both to completion, right? Is this useful to keep in?

Reply:
Yes, this avoids that other runs are aborted if one of them failed. I always found that confusing and normally I want to see all the (possible) errors at once.

Reply:
makes sense.